File: solutions/habitable_exoplanets.py
import numpy as np

input_str = str(input())
input_list = input_str.split()
problem = [float(number) for number in input_list]

L_star = problem[0]
r = problem[1]

if r < np.sqrt(L_star/1.1):
    print('too hot')
elif r > np.sqrt(L_star/0.53):
    print('too cold')
else:
    print('just right')

File: solutions/julia/habitable-exoplanets.jl
function habitable_exoplanet(L, r)
    rᵢ = √(L / 1.11)  # [AU]
    rₒ = √(L / 0.53)  # [AU]

    if r < rᵢ
        return "too hot"
    elseif r > rₒ
        return "too cold"
    else
        return "just right"
    end
end
